templates/rust-esp32-rust-board/diagram.json

```
@@ -29,5 +29,6 @@
     ],
     "serialMonitor": {
         "display": "auto"
```

@urish would you recommend using the following over `auto`?
```suggestion
    "serialMonitor": {
        "display": "terminal",
        "convertEol": true
```

---

If there's any fancy formatting (e.g. colors, etc), terminal usually looks better
"auto" uses an Arduino-compatible Serial Monitor which does not support ASCII escape codes